server/libs/domain/src/api-key/api-key.service.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { Test } from '@nestjs/testing';
+import { APIKeyEntity } from '@app/infra';
+import { ADMIN_AUTH_DTO, ADMIN_ENTITY } from '../../test';
+import { TestModule } from '../../test/test.module';
+import { ICryptoRepository } from '../auth';
+import { IKeyRepository } from './api-key.repository';
+import { APIKeyService } from './api-key.service';
+import { BadRequestException, UnauthorizedException } from '@nestjs/common';
+
+const ADMIN_KEY = Object.freeze({
```

Comment:
I think for consistency, for mock objects we should use camelCase (`ADMIN_KEY ` looks like it would hold a string). Maybe something like `apiKeyEntityMock`